src/PrestaShopBundle/Controller/Admin/Configure/ShopParameters/SearchAliasController.php

```diff
@@ -0,0 +1,246 @@
+
+            if (null !== $formHandlerResult->getIdentifiableObjectId()) {
+                $this->addFlash('success', $this->trans('Successful creation.', [], 'Admin.Notifications.Success'));
+
+                return $this->redirectToRoute('admin_search_alias_index');
+            }
+        } catch (Exception $e) {
+            $this->addFlash('error', $this->getErrorMessageForException($e, $this->getErrorMessages($e)));
+        }
+
+        return $this->render('@PrestaShop/Admin/Configure/ShopParameters/Search/form.html.twig', [
+            'form' => $form->createView(),
+            'help_link' => $this->generateSidebarLink($request->attributes->get('_legacy_controller')),
+            'enableSidebar' => true,
+            'layoutTitle' => $this->trans('New aliases', [], 'Admin.Shopparameters.Feature'),
+        ]);
+    }
+
+    #[AdminSecurity("is_granted('update', request.get('_legacy_controller'))", redirectRoute: 'admin_search_alias_index', message: 'You need permission to edit this.')]
+    public function editAction(
+        string $searchTerm,
+        Request $request,
+        #[Autowire(service: 'prestashop.core.form.identifiable_object.builder.alias_search_term_form_builder')]
+        FormBuilderInterface $formBuilder,
+        #[Autowire(service: 'prestashop.core.form.identifiable_object.builder.alias_search_term_form_handler')]
+        FormHandlerInterface $formHandler,
+    ): Response {
+        try {
+            /**
+             * @var AliasForEditing $editableAlias
+             */
+            $editableAlias = $this->dispatchQuery(new GetAliasesBySearchTermForEditing($searchTerm));
+
+            $form = $formBuilder->getFormFor($searchTerm);
+            $form->handleRequest($request);
+            $result = $formHandler->handleFor($searchTerm, $form);
+
+            if ($result->isSubmitted() && $result->isValid()) {
+                $this->addFlash('success', $this->trans('Successful update', [], 'Admin.Notifications.Success'));
+
+                return $this->redirectToRoute('admin_search_alias_index');
+            }
+        } catch (Exception $e) {
+            $this->addFlash('error', $this->getErrorMessageForException($e, $this->getErrorMessages($e)));
+        }
+
+        return $this->render('@PrestaShop/Admin/Configure/ShopParameters/Search/form.html.twig', [
+            'form' => $form->createView(),
+            'help_link' => $this->generateSidebarLink($request->attributes->get('_legacy_controller')),
+            'enableSidebar' => true,
+            'layoutTitle' => $this->trans(
+                'Edit aliases for %s',
+                [
+                    $editableAlias->getSearchTerm(),
+                ],
+                'Admin.Shopparameters.Feature'
+            ),
+        ]);
+    }
+
+    #[AdminSecurity("is_granted('read', request.get('_legacy_controller'))")]
+    public function searchAliasesForAssociationAction(Request $request): JsonResponse
```

Comment:
I'm not sure that this one is useful in the end, I think it was added in anticipation for a search input but since the UX evolved if it's used nowhere we can remove it